app/services/experiments/toggle.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Experiments::Toggle
+  include Service::Base
+
+  policy :current_user_is_admin
```

Comment:
Might be worth adding a contract to see if the setting exists? What does `set_and_log` do with a bogus setting?